AzureAuth.lutconfig

```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>true</ParallelTestRuns>
+  <TestCaseTimeout>30000</TestCaseTimeout>
+</LUTConfig>
```

Comment:
Is the LUT config used for Live Unit Testing configurations?

Reply:
I'm pretty sure that's the case, but IDK if it needs to be checked in. We haven't done that so far...

Reply:
If you do not have this file, Visual Studio will ask you to complete the configuration every time you start Live UT. Having it checked in just makes it easier for new devs, or for yourself on a new machine.